invenio_app_rdm/theme/assets/templates/search/ResultsGridItemTemplate.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Card, Image } from 'semantic-ui-react';
+import _truncate from 'lodash/truncate';
+
+export function ResultsGridItemTemplate(result, index) {
+    return (
+      <Card fluid key={index} href={`#`}>
+        <Card.Content>
+          <Card.Header>{result.metadata.title}</Card.Header>
```

Comment:
This changes with the new datamodel, just be aweare (or wait till its merged). It should be something like:

``` js
result.metadata.titles[0].title
```

Reply:
Now the module is released, so if you rebase on master, this will actually **not work** :) need to update (suggestion above)

Reply:
Right, that's the situation we will get in with model and use of model being in separate modules...

Maybe a facade over the returned results is needed so that we can have an interface to the result that will be stable across version changes? Something like defining a javascript model for a search result that is fed from the search API. It would present a uniform interface in terms of fields/methods to JSX usage irrespective of the changes in the serialization: it would be duplication, but it would serve the practical goal of resilience. It would be coded to present defaults if search API changes to ensure resilience. (sort of the idea for Ember models I think)
Something to consider maybe if makes sense.